packages/components/src/Clickable/Clickable.module.css

```diff
@@ -68,31 +68,31 @@
     --button-primary-color: var(--eds-color-alert-300);
   }
 }
-  
+
 .colorNeutral {
-  --button-primary-color: var(--eds-color-neutral-600);
+  --button-primary-color: var(--eds-color-neutral-500);
   --button-secondary-color: var(--eds-color-white);
   --button-tertiary-color: var(--eds-color-neutral-100);
 
   &:hover,
   &.stateHover,
   &:focus,
   &.stateFocus {
-    --button-primary-color: var(--eds-color-neutral-700);
+    --button-primary-color: var(--eds-color-neutral-600);
   }
 
   &:active,
   &.stateActive {
     /* TODO: Replace with neutral active color */
-    --button-primary-color: var(--eds-color-neutral-700);
```

Comment:
Can we leave this one as 700? The active state is supposed to be darker than the focus state, but it's currently the same. Dropping the other states down a level actually allows us to have the same value increase between these two states as the other hues.

Reply:
yeah I wavered on this one, but makes sense 👍